### ISSUE TITLE
A11y_mssql for VSCode_mssql Extension_Select top 1000 Results window_DraggingMovements: Unable to adjust the column width of the table using keyboard

### ISSUE THREAD

**Issue by aniketr99** [microsoft/vscode-mssql#17883]
## GitHubTags:
#A11yTCS;#Win11;#DesktopApp;#BM_mssql for VSCode_Win32_Feb2024;#A11ySev2;#mssql for VSCode;#WCAG2.5.7;#A11yMAS;#Win32;#AINotInScope;#Keyboard;A11yWCAG2.2;#Dragging Movements;

## Environment Details:
Application Name: mssql for VSCode
Operating System: Win11[23H2]
OS Build: 26016.1012
Visual studio code: 1.86.2
Extension Version: v1.22.1

## Repro Steps:

1. Open Visual studio code
2. Tab till extension button and search for "mssql".
3. Tab till install and move to the SQL Server button.
4. Tab till Add Connection Button and insert the valid credentials.
5. Tab till the database and hit enter.
6. Tab till the System database and hit enter.
7. Tab till the Tables and hit enter.
8. Tab till any table under System tables and hit shift + F10
9. Tab till Select Top 1000 and hit enter
10. Tab till Execute Query and hit enter
11. Tab till "Results" dropdown and hit enter
12. Tab till grid and verify whether column width is adjustable or not.

## Actual:
 Unable to adjust the column width of the table using keyboard.

## Expected:
Users should be able to adjust the column width of the table using keyboard.

## User Impact:
Keyboard users will not be able to adjust the width according to their convenience.

[“Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

## Attachment:
![ppt2CB pptm  -  AutoRecovered2](https://github.com/microsoft/vscode-mssql/assets/100269580/67e2fafd-6634-4e7c-b7b6-24027517b02d)

**Comment by aniketr99** [comment]
#Regressed:08-12-24;
Regressed the bug in the latest environment, there is no resize option provided in the grid. hence the issue is not fixed. 
![image](https://github.com/user-attachments/assets/4749a885-004b-488e-846f-4f94dc7b2125)

Expected result: The column width should be adjustable the column headers by using the resize option, that users can resize the column to fit their needs and preferences. This allows for better accessibility and usability.

**Comment by kburtram** [comment]
@laurennat it looks like the tester said this issue isn't fixed. Could you please follow up to get this issue resolved?

**Comment by laurenastrid1** [comment]
This has also been fixed, i just need a tester to verify.
![Image](https://github.com/user-attachments/assets/4d385f87-d5da-4ce2-bf0f-4429dea8d178)

**Comment by scottaohara** [comment]
the ui to adjust the column size definitely meets the goal, it's just that the UI to adjust the column size also be accessible via a pointer interaction.  would this be possible to invoke via a right-click action, or via a menu item in a right-click context menu for the column or just the column header ?

**Comment by ManishaC1** [comment]
@scottaohara 

The UI to adjust the column size is accessible only using keyboard shortcut key Ctrl+Shift and not via right-click action.

**Comment by Roopesh-T** [comment]
reducing to sev3 as the headers are getting adjusted with a double click on the resizer.
#A11ysev3

**Comment by ManishaC1** [comment]
@laurenastrid1 

Can you please close this bug, as this bug is flowing in sev2.

We have logged the following bug under Sev3. Please track this bug
Bug link: https://github.com/microsoft/vscode-mssql/issues/19079

**Comment by laurenastrid1** [comment]
I am working on it now!

**Comment by laurenastrid1** [comment]
New Bug being tracked here: https://github.com/microsoft/vscode-mssql/issues/19079
